crates/viewer/re_viewport/src/viewport_ui.rs

```diff
@@ -585,16 +585,6 @@ impl<'a> egui_tiles::Behavior<ViewId> for TilesDelegate<'a, '_> {
         visuals.panel_fill.gamma_multiply(0.5)
     }
 
-    /// When drag-and-dropping a tile, the candidate area is drawn with this stroke.
-    fn drag_preview_stroke(&self, _visuals: &egui::Visuals) -> egui::Stroke {
-        egui::Stroke::new(1.0, egui::Color32::WHITE.gamma_multiply(0.5))
-    }
-
-    /// When drag-and-dropping a tile, the candidate area is drawn with this background color.
-    fn drag_preview_color(&self, _visuals: &egui::Visuals) -> egui::Color32 {
-        egui::Color32::WHITE.gamma_multiply(0.1)
-    }
-
```

Comment:
TODO: revert this. I thought they were unused methods, but I see now they are trait-overrides